src/pushNotifications/PermissionHandling.ts

```diff
@@ -0,0 +1,112 @@
+import * as Permissions from 'expo-permissions';
+import * as Notifications from 'expo-notifications';
+import Constants from 'expo-constants';
+import { Alert } from 'react-native';
+import { addToStore, readFromStore } from '../helpers';
+import { PermissionStatus } from 'expo-permissions';
+import { device, texts } from '../config';
+import { handleIncomingToken, PushNotificationStorageKeys } from './TokenHandling';
```

Comment:
here i found another case, where imports are mixed and not libraries first and local stuff second, divided by a line break.
what do you think about grouping all expo imports and pulling `from 'react-native` to the top?
in most cases we import stuff from prop types first, react second, react native third. do you think this is worth it?

```
import { Alert } from 'react-native';
import * as Permissions from 'expo-permissions';
import * as Notifications from 'expo-notifications';
import Constants from 'expo-constants';
import { PermissionStatus } from 'expo-permissions';

import { addToStore, readFromStore } from '../helpers';
import { device, texts } from '../config';
import { handleIncomingToken, PushNotificationStorageKeys } from './TokenHandling';
```

Reply:
updated it in 94bd1238af679bc33b72c53861317369ad81796d

we can do it if you prefer it that way. or are there any eslint rules/plugins that can sort everything neatly for us?

Reply:
my experience is, that it helps having same structures across different projects. you can scan files quickly this way.
i do not know about an eslint solution for this, maybe there is one nowadays? research needed.

Reply:
opened an issue for that: https://github.com/ikuseiGmbH/smart-village-app-app/issues/128